search-sentiments-map/public/index.html

```diff
@@ -64,33 +64,44 @@ <h4 class="modal-title" id="modal-title"></h4>
     <div id="top-left-div">
       <h3 id="topic-header">Worldwide sentiments</h3>
       <label class="switch">
-        <input id="sentiment-popularity-check" type="checkbox" onclick="loadCountryDataByMode()">
+        <input id="sentiment-popularity-check" type="checkbox" onclick="loadRegionDataByMode()">
         <span class="slider"></span>
       </label>
-      <span id="switch-info">Switch between Sentiment Mode & Popularity Mode.</span>
+      <span id="switch-info">Switch between <span class="mode-span" data-toggle="popover" data-trigger="hover" data-content="Shows worldwide sentiments on selected topic" data-placement="top">Sentiment Mode</span> 
+          & <span class="mode-span" data-toggle="popover" data-trigger="hover" data-content="Shows worldwide interest on selected topic" data-placement="top">Popularity Mode</span>.</span>
     </div>
 
-    <div id="right-div">
+    <!-- The inner text is a dash, which is longer than a hyphen. -->
```

Comment:
Is this necessary?

Reply:
Yes because what we type with the keyboard is usually a hyphen